src/NzbDrone.Core/Notifications/Email/EmailSettings.cs

```diff
@@ -46,6 +46,9 @@ public EmailSettings()
         [FieldDefinition(6, Label = "Recipient Address")]
         public string To { get; set; }
 
+        [FieldDefinition(7, Label = "Attach Books", HelpText = "Add books as an attachment on import", Type = FieldType.Checkbox)]
```

Comment:
Want to go ahead and change recipient to enumerable and add cc and bcc fields from Radarr PR?